source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -46,6 +46,21 @@ def _get_shouldCreateTreeInterceptor(self):
 class Application(Document):
 	shouldCreateTreeInterceptor = False
 
+class WebDialog(Document):
+	"""
+	A dialog that will use a treeInterceptor if its parent currently does.
+	"""

```

Comment:
It might be good to explain the practical impact in real world terms; i.e. that web dialogs should use browse mode by default unless they're inside an ARIA application.